### PR TITLE
Entry for foo

### PR DESCRIPTION
**Pull request by codestreamtest**
To test the things

### CODE REVIEW

**Comment by codestreamtest** [comment]
not in cs

```
		this._versionFormatted = agentOptions.extension.versionFormatted;

```
								
[freezer.js](https://github.com/codestreamtest/beerfridge/blob/9c93c42a6d95ffb38f5ece44c0d955d0bde9317f/freezer.js#L33) (Line 33)

File: freezer.js [hunk 1]
```diff
@@ -18,6 +18,7 @@ import { SelectionDecorationProvider } from "./providers/selectionDecorationProv
 import { SetServerUrlRequestType} from "./protocols/agent/agent.protocol";
 // import { WebviewSidebarActivator } from "./views/webviewSidebarActivator";
 
+// entry point test
```
Comment (planteater) [inline freezer.js:21]: single again

Comment (planteater) [reply freezer.js:21]: reply to single

**Comment by planteater** [comment]
tip top

**Pull request closed without merging**